azure-pipelines.yml

```diff
@@ -1,30 +1,54 @@
-# .NET Desktop
-# Build and run tests for .NET Desktop or Windows classic desktop solutions.
-# Add steps that publish symbols, save build artifacts, and more:
-# https://docs.microsoft.com/vsts/pipelines/apps/windows/dot-net
+# ASP.NET Core
+# Build and test ASP.NET Core projects targeting .NET Core.
+# Add steps that run tests, create a NuGet package, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/dotnet-core
 
+# the build will trigger on any changes to the master branch
+trigger:
+- master
+
+# the build will run on a Microsoft hosted agent, using the lastest Windows VM Image
 pool:
-  vmImage: 'VS2017-Win2016'
+  vmImage: 'windows-latest'
 
+# these variables are available throughout the build file
+# just the build configuration is defined, in this case we are building Release packages
 variables:
-  solution: '**/*.sln'
-  buildPlatform: 'Any CPU'
   buildConfiguration: 'Release'
 
+#The build has 3 seperate tasks run under 1 step
 steps:
-- task: NuGetToolInstaller@0
-
-- task: NuGetCommand@2
+    
+# The first task is the dotnet command build, pointing to our csproj file
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet build'
   inputs:
-    restoreSolution: '$(solution)'
+    command: 'build'
+    arguments: '--configuration $(buildConfiguration)'
+    projects: 'carwings.net/carwings.net.csproj'
 
-- task: VSBuild@1
+# The second task is dotnet pack command again pointing to the csproj file
+# The nobuild means the project will not be compiled before running pack, because its already built in above step
+- task: DotNetCoreCLI@2
+  displayName: "dotnet pack"
   inputs:
-    solution: '$(solution)'
-    platform: '$(buildPlatform)'
-    configuration: '$(buildConfiguration)'
+    command: 'pack'
+    arguments: '--configuration $(buildConfiguration)'
+    packagesToPack: 'carwings.net/carwings.net.csproj'
+    nobuild: true
+    versioningScheme: 'off'
 
-- task: VSTest@2
+# The last task is a nuget command, nuget push
+# This will push any .nupkg files to the 'FlexChargingPackages' artifact feed
```

Comment:
this probably shouldn't be here

Reply:
Hmmm, yeah, that should probably live in my branch only.